app/src/main/java/io/seqera/wave/cli/App.java

```diff
@@ -382,13 +384,21 @@ protected void validateArgs() {
         if( !isEmpty(platform) && !VALID_PLATFORMS.contains(platform) )
             throw new IllegalCliArgumentException(String.format("Unsupported container platform: '%s'", platform));
 
+        // check labels
+        if( labels!=null ) {
+            for( String it : labels) {
+                if( !isLabel(it) ) throw new IllegalCliArgumentException("Invalid docker label syntax - offending value: " + it);
```

Comment:
instead of `Invalid docker label syntax`, I would slightly edit to `Invalid image label syntax`

Reply:
done